src/board/board.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { Tile } from "../tile/tile";
+import {renderLine, Board, renderBoard, NullableTile} from "./board"
+
+describe("renderLine", () => {
+    it("Testing a line with both undefined and Tile values", () => {
```

Comment:
This is nitpicking, but I figured it could be helpful in the future:

The idea of the `it` methods is that they allow to name tests so that it reads like:

```ts
describe("renderLine", () => {
    // This test can be read as 'renderLine: it should render a line with blah blah blah'
    it("should render a line with blah blah blah", () => {
        // ...
    });
    // This test can be read as 'renderLine: it should not fail when blah blah blah'
    it("should not fail when blah blah blah", () => {
        // ...
    });
});
```

That's why we tend to name our tests starting with 'should...'.

But again, that's nitpicking. No need to change this now.

Reply:
I will take that into account when writing future tests. Thanks !